utils/isOwner.js

```diff
@@ -0,0 +1,15 @@
+const isOwner = async (collection, userId, id) => {
+  try {
+    const singleCollection = await collection.findOne({ id });
+
+    if(userId === singleCollection.user.id){
+      return true;
+    }
+
+    return false;
+  } catch(err) {
+    console.error(err);
```

Comment:
What will this function return in this case?

Reply:
Yeah I think it should return false